src/main/scala/org/apache/spark/sql/delta/catalog/DeltaCatalog.scala

```diff
@@ -94,6 +93,30 @@ class DeltaCatalog(val spark: SparkSession) extends DelegatingCatalogExtension
     loadTable(ident)
   }
 
+  private def isPathIdentifier(ident: Identifier): Boolean = {
+    try {
+      ident.namespace().sameElements(Array("delta")) && new Path(ident.name()).isAbsolute
```

Comment:
I've definitely overlooked this before, but does the catalog name "delta" need to itself be case insensitive? (This is more a data source V2 decision than a Delta one, I think, but I don't really have strong opinions on which way is right.)

Reply:
that's a great question. I feel like it should also even support `org.apache.spark.delta.DeltaDataSource` too, but no one even writes that. It probably should be case sensitive depending on the case sensitivity flag (insensitive by default)